vibe_code_apps_20/wattlab/notebooks/twin_cascade.py

```
@@ -0,0 +1,188 @@
+        simulate(patched, epw, out)
+        ann = annual_from_output_dir(
+            out, elec_rate_usd_per_kwh=elec_rate, gas_rate_usd_per_therm=gas_rate
+        )
+        records.append(
+            build_result_record(
+                run_id=f"{run_id}_{mid}",
+                measure_id=mid,
+                idf_path=patched,
+                annual=ann,
+                artifacts=[str(out / "eplustbl.htm"), str(patched)],
+                extra_flags=list(meta.get("flags") or []) + ["openfdd_wattlab"],
```

Ruff (0.15.21)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)